src/nasir/Exam2_Test.java

```diff
@@ -0,0 +1,39 @@
+package nasir;
+
+public class Exam2_Test {
+	int sum=6;	
+	 void findPairOfSum(int[] arr) {
```

Comment:
sum should be parameterized

Reply:
Point noted and I can do like this
![image](https://user-images.githubusercontent.com/96292117/148944463-8de8d302-c9bb-4386-8bd3-0110c3de751e.png)